scanner/scanlog/scanlog.go

```diff
@@ -54,6 +54,7 @@ var (
 	numWorkers    = flag.Int("num_workers", 2, "Number of concurrent matchers")
 	parallelFetch = flag.Int("parallel_fetch", 2, "Number of concurrent GetEntries fetches")
 	startIndex    = flag.Int64("start_index", 0, "Log index to start scanning at")
+	endIndex      = flag.Int64("end_index", 0, "Log index to end scanning at (non-inclusive, 0 - end of Log)")
```

Comment:
maybe `0 for end of log` or `0 = end of Log` ?

Reply:
Done.